hadoop-ozone/dist/src/main/smoketest/security/ozone-secure.robot

```diff
@@ -16,14 +16,44 @@
 *** Settings ***
 Documentation       Smoke test to start cluster with docker-compose environments.
 Library             OperatingSystem
+Library             String
 Resource            ../commonlib.robot
 
+*** Variables ***
+${ENDPOINT_URL}       http://s3g:9878
+
+*** Keywords ***
+Install aws cli s3 centos
+    Execute                    sudo yum install -y awscli
+    Execute                    sudo yum install -y krb5-user
+Install aws cli s3 debian
+    Execute                    sudo apt-get install -y awscli
+    Execute                    sudo apt-get install -y krb5-user
+
+Install aws cli
+    ${rc}              ${output} =                 Run And Return Rc And Output           which apt-get
+    Run Keyword if     '${rc}' == '0'              Install aws cli s3 debian
+    ${rc}              ${output} =                 Run And Return Rc And Output           yum --help
+    Run Keyword if     '${rc}' == '0'              Install aws cli s3 centos
+
+Setup credentials
+    ${hostname}=        Execute                    hostname
+    Execute             kinit -k testuser/${hostname}@EXAMPLE.COM -t /etc/security/keytabs/testuser.keytab
+    ${result} =         Execute                    ozone sh s3 getsecret
+    ${accessKey} =      Get Regexp Matches         ${result}     (?<=awsAccessKey=).*
+    ${secret} =         Get Regexp Matches	       ${result}     (?<=awsSecret=).*
```

Comment:
whitespace:tabs in line

Reply:
Can we remove this change from here? As this Jira purpose is not for this.

Reply:
+1 to move it to a separate Jira. Let's keep only ozoneManager -> om refactoring changes in this patch.

Reply:
removed s3 related changes, HDDS-1043 will have them.

Reply:
Still we have a few of them (commonawslib, docker-config)

Reply:
@elek  thanks for catching that, removed from both files (commonawslib, docker-config).